src/braket/default_simulator/openqasm/program_context.py

```diff
@@ -375,7 +376,7 @@ class ProgramContext:
 
     """
 
-    def __init__(self):
+    def __init__(self, circuit=Circuit()):
```

Comment:
Rather than having this in the base constructor, leave it to the subclasses.
```suggestion
    def __init__(self):
```

Reply:
Maybe this could be
```suggestion
    def __init__(self, circuit=None):
```

Reply:
Yep, don't want to have mutable defaults.